tests/testthat/test-library_call_linter.R

```diff
@@ -0,0 +1,51 @@
+test_that("library_call_linter skips allowed usages", {
+
+  expect_lint(c("library(dplyr)", "print('test')"),
+    NULL,
+    library_call_linter()
+  )
+
+  expect_lint("print('test')",
+    NULL,
+    library_call_linter()
+  )
+
+  expect_lint(c("# comment", "library(dplyr)"),
+    NULL,
+    library_call_linter()
+  )
+
+  expect_lint(c("print('test')", "# library(dplyr)"),
+    NULL,
+    library_call_linter()
+  )
+})
+
+test_that("library_call_linter warns on disallowed usages", {
```

Comment:
Maybe add a test that also checks the location metadata?

Reply:
I thought about it and should have done it to start with 😄.  I'll add it for those that return more than one lint.